docs/tutorial/part-two/index.md

```diff
@@ -295,39 +294,39 @@ export default () => (
 
-The finished page should now look like:
+Halaman yang telah selesai seharusnya terlihat seperti:
 
-![User list page with CSS modules](css-modules-userlist.png)
+![Halaman daftar User dengan CSS modules](css-modules-userlist.png)
 
 ### CSS-in-JS
 
-CSS-in-JS is a component-oriented styling approach. Most generally, it is a pattern where [CSS is composed inline using JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
+CSS-in-JS merupakan pendekatan styling berorientasi komponen. Kebanyakan pada umumnya, Hal itu merupakan pola dimana [CSS dibentuk secara _inline_ menggunakan JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
 
-#### Using CSS-in-JS with Gatsby
+#### Menngunakan CSS-in-JS dengan Gatsby
 
-There are many different CSS-in-JS libraries and many of them have Gatsby plugins already. We won't cover an example of CSS-in-JS in this initial tutorial, but we encourage you to [explore](/docs/styling/) what the ecosystem has to offer. There are mini-tutorials for two libraries, in particular, [Emotion](/docs/emotion/) and [Styled Components](/docs/styled-components/).
+Ada banyak pustaka CSS-in-JS berbeda dan kebanyakan sudah memiliki plugin Gatsby. Kita tidak akan membahas contoh CSS-in-JS pada tutorial awal ini, tapi kita menganjurkan Anda untuk [menjelajahi](/docs/styling/) apa yang ekosistem tawarkan. Ada tutorial mini untuk dua pustaka, khususnya, [Emotion](/docs/emotion/) dan [Styled Components](/docs/styled-components/).
 
-#### Suggested reading on CSS-in-JS
+#### Bacaan yang disarankan tentang CSS-in-JS
 
-If you're interested in further reading, check out [Christopher "vjeux" Chedeau's 2014 presentation that sparked this movement](https://speakerdeck.com/vjeux/react-css-in-js) as well as [Mark Dalgleish's more recent post "A Unified Styling Language"](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660).
+Jika Anda tertarik bacaan lebih lanjut, telusuri [Christopher "vjeux" Chedeau's 2014 presentation that sparked this movement](https://speakerdeck.com/vjeux/react-css-in-js) dan juga [Mark Dalgleish's more recent post "A Unified Styling Language"](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660).
 
-### Other CSS options
+### Opsi CSS lainnya
 
-Gatsby supports almost every possible styling option (if there isn't a plugin yet for your favorite CSS option, [please contribute one!](/contributing/how-to-contribute/))
+Gatsby mendukung hampir setiap opsi styling (jika belum ada plugin untuk opsi CSS favorit Anda, [silahkan berkontribusi!](/contributing/how-to-contribute/))
 
 - [Typography.js](/packages/gatsby-plugin-typography/)
 - [Sass](/packages/gatsby-plugin-sass/)
 - [JSS](/packages/gatsby-plugin-jss/)
 - [Stylus](/packages/gatsby-plugin-stylus/)
 - [PostCSS](/packages/gatsby-plugin-postcss/)
 
-and more!
+and banyak lagi!
```

Comment:
And banyak lagi should be “dan banyak lagi”

Reply:
got it